www/inc/playerlib.php

```diff
@@ -409,32 +400,25 @@ function genLibrary($flat) {
 function genLibraryUTF8Rep($flat) {
 	$lib = array();
 
-	// use Artist or AlbumAetist for the Artist column
-	$libartist = $_SESSION['libartistcol'];
```

Comment:
@moodeaudio I removed `libartistcol` here and in the js, but I'm not sure if there is more cleanup to do (db migration or something)